contracts/contracts/token/OUSD.sol

```diff
@@ -234,32 +256,89 @@ contract OUSD is Initializable, InitializableERC20Detailed, Governable {
         address _to,
         uint256 _value
     ) internal {
-        bool isNonRebasingTo = _isNonRebasingAccount(_to);
-        bool isNonRebasingFrom = _isNonRebasingAccount(_from);
-
-        // Credits deducted and credited might be different due to the
-        // differing creditsPerToken used by each account
-        uint256 creditsCredited = _value.mulTruncate(_creditsPerToken(_to));
-        uint256 creditsDeducted = _value.mulTruncate(_creditsPerToken(_from));
+        if (_from == _to) {
```

Comment:
I think we can try removing this and seeing if anything bad happens. As it is it would allow massive self transfers to succeed, which would be weird.

Reply:
that is a good catch: https://github.com/OriginProtocol/origin-dollar/pull/2298/commits/108f0ebac7abb80d0c7dadd012b4c389d86fd1cc